README.md

```diff
@@ -51,6 +52,45 @@ Runtime environments for serverless cloud computing for multiple coding language
 | Node.js | 16.0    | [open-runtimes/node.js:16.0](https://hub.docker.com/r/open-runtimes/node.js) | [Node.js Examples]() | [![Docker Pulls](https://img.shields.io/docker/pulls/open-runtimes/node.js?color=f02e65&style=flat-square)](https://hub.docker.com/r/open-runtimes/node.js) |
 | PHP     | 8.0     | [open-runtimes/php:8.0](https://hub.docker.com/r/open-runtimes/php)      | [PHP Examples]() | [![Docker Pulls](https://img.shields.io/docker/pulls/open-runtimes/php?color=f02e65&style=flat-square)](https://hub.docker.com/r/open-runtimes/php) |
 
+## Runtimes introduction
+
+All runtimes share the same folder structure, but each also adds runtime-specific files to properly support it's package manager.
+
```

Comment:
I recommend using the `tree ` command within a runtime folder to give a better representation

```
.
├── build.sh
├── docker-compose.yml
├── Dockerfile
├── example
├── example.sh
├── launch.sh
├── README.md
└── server.X

1 directory, 9 files
```

After this, you can have bullet points to explain the function of each file/ directory 

- `example/` - This folder contains a sample cloud function that you can use to test the runtime. 
- `Dockerfile` -  Defines how to build the runtime with instructions to prepare the environment for the runtime. These images are usually based on official Alpine or Ubuntu images.

... And so on for the other files and folders.

Reply:
Added folder structure